glide.yaml

```diff
@@ -12,7 +12,7 @@ import:
   version: 763351934333e34370292b308c74bcb71035e22a
   repo: https://github.com/src-d/go-billy-siva.git
 - package: gopkg.in/src-d/go-git.v4
-  version: 773841734e11f0f607f6b047235f8527d475538f
+  version: 7aa9d15d395282144f31a09c0fac230da3f65360
```

Comment:
Since this commit is tagged in go-git as "v4.0.0-rc14", maybe it's better use the tag name instead the commit here so you can see which go-git version is being used at a glance.

What do you think?

Reply:
I'm not sure if the tag is reachable from `gopkg.in`